mvpa2/support/afni/lib_prep_afni_surf.py

```diff
@@ -286,19 +311,75 @@ def yesno2bool(d, k): # dict, key
             c['outvol_ext'] = '+' + c['outvol_view']
             c['outvol_fullext'] = c['outvol_ext'] + '.HEAD'
 
-
     return c
 
+
+
+def _ensure_expvol_is_plump(expvol):
+    '''Helper function that raises an error if the experimental volume
+        is oblique, because such volumes are not supported (yet)'''
+    cmd = '3dinfo "%s"' % expvol
+    info = utils.cmd_capture_output(cmd)
```

Comment:
alternatively (and ideally, since then wouldn't even depend on having afni), this function ( cmd_capture_output) could be mocked while could just be mocked while running a test and provide coverage.  We used mock  already in 1 spot, so should be ok to use testing this one as well (just conditional on having mock module).  Should be smth like
```python

def _test_ensure_expvol_is_plump():
    # actual test function
    ret = _ensure_expvol_is_plump(SOMEVOL)
    # do tests

def test_ensure_expvol_is_plump():
     # first lets try to run as is if 3dinfo is available, so we verify 'integration' with existing afni
     if externals.exists('3dinfo'):
          yield _test_ensure_expvol_is_plump
     # now mocked version... if test runs awhile could be in `else:` I guess
     if externals.exists('mock'):
         import mock
         with patch.object(utils, 'cmd_capture_output', return_value=DESIRED_RETURN):
               yield _test_ensure_expvol_is_plump
```

Then this would test it with and/or without afni installed ;)